javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  
+    studentName === "" ? console.log("Please add a name!"):undefined;
+
+    if (studentName == "Queen"  || studentName === "queen"){
```

Comment:
- I added "else" to follow up on the previous comment, so it is more consistent where it will no longer approve adding an empty name

- Use strict equality (===) for both comparisons in the condition if (studentName == "Queen" || studentName === "queen") for consistency.

- and you can simply use toLowerCase() method.


```suggestion
      else if (studentName.toLowerCase() === "queen") {
```